Tests/UnitTests/EdgeEndpointTests.swift

```diff
@@ -56,49 +57,59 @@ class EdgeEndpointTests: XCTestCase {
         XCTAssertEqual(.production, EdgeEnvironmentType(optionalRawValue: "invalid"))
     }
 
-    func testDefaultProductionEndpoint() {
-        let endpoint = EdgeEndpoint(type: .production)
-        let expected = "https://\(EdgeConstants.NetworkKeys.EDGE_DEFAULT_DOMAIN)\(EdgeConstants.NetworkKeys.EDGE_ENDPOINT_PATH)"
-        XCTAssertEqual(expected, endpoint.endpointUrl)
-    }
+    // MARK: EdgeEndpoint tests
+    func testEdgeEndpoint_interact_defaultDomain() {
+        // cases defined as: ($0: input EnvironmentType, $1: expected output EdgeEndpoint URL, $2: Test case name)
+        let cases = [(EdgeEnvironmentType.production, "https://edge.adobedc.net/ee/v1/interact", "DefaultProductionEndpoint"),
```

Comment:
This will work better if the test case doesn't fail after the first failure. Looking at the other tests in this test suite, it looks like you can set `continueAfterFailure = true`, but you should double check. I'd suggest explicitly setting the flag to true with a comment stating it allows these tests with multiple parameters to run each case even when one fails, so someone later doesn't set it to false.